python/nx-cugraph/pyproject.toml

```
@@ -90,7 +89,7 @@ matrix-entry = "cuda_suffixed=true"
 
 [tool.black]
 line-length = 88
-target-version = ["py39", "py310", "py311"]
+target-version = ["py310", "py310", "py311"]
```

Why is `py310` listed here twice?

---

```suggestion
target-version = ["py310", "py311"]
```

oh wow, thank you! Think my `rapids-reviser` code did not expect this to ever include more than one value. But sure enough, it can (and in fact the `black` docs even say it *should*): https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#t-target-version